packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-draggable-dummy-area.tsx

```diff
@@ -0,0 +1,58 @@
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
+
+import { BlankDraggableDummyAnswer } from './blank-draggable-dummy-answer'
+import type { FillInTheBlanksExerciseProps } from '..'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
+interface BlankDraggableDummyAreaProps {
+  additionalDraggableAnswers: FillInTheBlanksExerciseProps['state']['additionalDraggableAnswers']
+}
+
+export function BlankDraggableDummyArea(props: BlankDraggableDummyAreaProps) {
+  const { additionalDraggableAnswers } = props
+
+  const blanksExerciseStrings = useEditorStrings().plugins.blanksExercise
+
+  const dummyDraggables = additionalDraggableAnswers.map(
+    ({ answer }) => answer.value
+  )
+
+  function handleDummyAnswerAdd() {
+    additionalDraggableAnswers.insert()
+  }
+
+  function handleRemoveDummyAnswer(index: number) {
+    additionalDraggableAnswers.remove(index)
+  }
+
+  function handleDummyAnswerChange(text: string, index: number) {
+    additionalDraggableAnswers[index].answer.set(text)
+  }
+
+  return (
+    <div className="mt-8 px-4">
+      {blanksExerciseStrings.dummyAnswers}:
+      <div className="flex min-h-8 flex-wrap items-stretch">
+        {dummyDraggables.map((answer, index) => (
+          <BlankDraggableDummyAnswer
+            key={index}
+            text={answer}
+            onChange={(event) => {
+              handleDummyAnswerChange(event.target.value, index)
+            }}
+            onRemove={() => {
+              handleRemoveDummyAnswer(index)
+            }}
+          />
+        ))}
+      </div>
```

Comment:
Extra space removed, thanks for pointing that out. The title is there in the design, even without any extra incorrect answers, so it would be great if you could pass that feedback to Gregor as well.

Reply:
Implemented this suggestion and some other minor fixes in https://github.com/serlo/frontend/pull/3310

Will share it with Gregor as well and see what he thinks.